libs/npm/core/src/hooks/api/subscriber/useApiSubscriberReports.ts

```diff
@@ -101,7 +101,7 @@ export const useApiSubscriberReports = (
     },
     addContentToReport: (reportId: number, content: IReportInstanceContentModel[]) => {
       return api.post<never, AxiosResponse<IReportModel>, any>(
-        `/subscriber/reports/${reportId}/content`,
+        `/subscriber/reports/${reportId}/content/fast`,
```

Comment:
For this to work you will need to publish a new version of the tno-core package and add it to both webapps.